css-two-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Karla, sans-serif;
+}
+
+.container {
+  width: 50%;
+  margin: 0 auto;
+  padding-top: 50px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  line-height: 1.5;
+}
+
+.section {
+  display: flex;
+}
+
+.name-and-description {
+  flex-basis: 60%;
+  margin-top: 30px;
+}
+
+.icon {
+  flex-basis: 25%;
+  width: 100%;
+  align-self: flex-start;
+}
+
+.heading {
+  text-transform: uppercase;
+  font-size: 2.8em;
+  letter-spacing: 0.05em;
+  margin: -5px;
```

Comment:
Avoid negative margin or padding, they are a gateway to terrible things 😬

Reply:
Turns out that I just had to set the margin to 0 to get the effect I wanted 🤣 Cool!